rules/luma.js

```diff
@@ -1,4 +1,7 @@
+const postcssLess = require("postcss-less");
+
 module.exports = {
+  customSyntax: postcssLess,
```

Comment:
What does this do? 😏

Reply:
Dit moet van de nieuwe stylelint, is niet meer een default van hun.
Dit is zodat hij weet welke postcss parser nodig is